src/content/docs/blog/ses-1.6.0.md

```diff
@@ -0,0 +1,295 @@
+    __options__: true,
+    modules: {
+      'my-module-specifier': {
+        source: new ModuleSource(`
+            export default 42;
+          `),
+      },
+    },
+  });
+  console.log(compartment.importNow('my-module-specifier').default); // 42
+  ```
+
+- Prior to version 1.6.0, a module namespace object could be used in place
+  of a module descriptor.
+  Particularly, the `Compartment` implementation included a `module` method
+  that could be used to acquire the module namespace object of a module that
+  has not yet been loaded, such that a module namespace could be used to link
+  compartments or settle cyclic dependencies.
+  This practice is deprecated now in favor of using a `namespace` module
+  descriptor.
+
+  ```js
+  // before:
+  const c1 = new Compartment({}, {
+    'c1-module-specifier': new StaticModuleRecord(`
+      export default 42;
+    `)
+  });
+  const c2 = new Compartment({}, {
+    'c2-module-specifier': c1.module('c1-module-specifier'),
+  });
+  ```
+
+  ```js
+  // after
+  const c1 = new Compartment({
+    modules: {
+      'c1-module-specifier': new ModuleSource(`
+        export default 42;
+      `)
+    },
+  });
+  const c2 = new Compartment({
+    __options__: true,
+    modules: {
+      'c2-module-specifier': {
+        namespace: 'c1-module-specifier',
+        compartment: c1,
+      },
+    },
+  });
+  ```
+
+  Module descriptors with the `source` key will construct a fresh instance
+  of the source in this compartment.
+  Module descriptors with the `namespace` key will link to a module instance in
+  the designated compartment.
+
+- Prior to version 1.6.0, a module descriptor with `record` property
+  could be used to instantiate a module with a different `import.meta.url` than
+  the associated module specifier.
+  These descriptors should migrate to `namespace` module descriptors and
+  must explicitly designate the `compartment` property if they previously
+  relied on the default `compartment`.
+  The default compartment for `namespace` and `source` descriptors is the
+  "parent compartment": the compartment for which `Compartment` is
+  the initial, intrinsic `compartment.globalThis.Compartment`.
+  The default compartment for `record` descriptors is child compartment.
+
+  ```js
+  // before:
+  const compartment = new Compartment(
+    {},
+    {
+      'submodule/dependency': new StaticModuleRecord(`
+        export default 42;
+      `),
+      dependant: {
+        record: new StaticModuleRecord(`
+          import meaning from 'dependency';
+          export default meaning;
+        `),
+        specifier: 'submodule/depenant',
+      },
+    },
+    {
+      resolveHook(importSpecifier, referrerSpecifier) {
+        const path = referrerSpecifier.split('/');
+        path.pop();
+        path.push(...importSpecifier.split('/'));
+        return path.join('/');
+      },
+    },
+  );
+  console.log(compartment.importNow('dependant').default); // 42
+  ```
+
+  ```js
+  // after
```

Comment:
I'm not seeing a relevant difference between this block and its predecessor... what am I missing?